waterlog/src/Stylesheets/_alerts.scss

```diff
@@ -0,0 +1,46 @@
+
+table {
+  font-family: "Trebuchet MS", Arial, Helvetica, sans-serif;
```

Comment:
do tables and h3 tags purposefully have a different font to the rest? If not, consider declaring the font-family in the main stylesheet on the body tag to have it applied everywhere